The job **`dist-arm-linux`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/11175332209/job/31066517186) [(plain)](https://github.com/rust-lang-ci/rust/commit/a94af5757a602b7f2dcc0b9eb1885ffbbcfa05b7/checks/31066517186/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
[RUSTC-TIMING] build_script_build test:false 0.242
   Compiling compiler_builtins v0.1.131
The following warnings were emitted during compilation:

warning: compiler_builtins@0.1.131: In file included from /checkout/src/llvm-project/compiler-rt/lib/builtins/fp_lib.h:24,
warning: compiler_builtins@0.1.131:                  from /checkout/src/llvm-project/compiler-rt/lib/builtins/multc3.c:14:
warning: compiler_builtins@0.1.131: /checkout/src/llvm-project/compiler-rt/lib/builtins/multc3.c: In function ‘__multc3’:
warning: compiler_builtins@0.1.131: /checkout/src/llvm-project/compiler-rt/lib/builtins/int_math.h:71:32: error: implicit declaration of function ‘__builtin_copysignq’; did you mean ‘__builtin_copysign’? [-Werror=implicit-function-declaration]
warning: compiler_builtins@0.1.131:    71 | #define crt_copysignf128(x, y) __builtin_copysignq((x), (y))
warning: compiler_builtins@0.1.131:       |                                ^~~~~~~~~~~~~~~~~~~
warning: compiler_builtins@0.1.131: /checkout/src/llvm-project/compiler-rt/lib/builtins/fp_lib.h:394:24: note: in expansion of macro ‘crt_copysignf128’
warning: compiler_builtins@0.1.131:   394 | #define crt_copysigntf crt_copysignf128
warning: compiler_builtins@0.1.131: /checkout/src/llvm-project/compiler-rt/lib/builtins/multc3.c:33:11: note: in expansion of macro ‘crt_copysigntf’
warning: compiler_builtins@0.1.131: /checkout/src/llvm-project/compiler-rt/lib/builtins/multc3.c:33:11: note: in expansion of macro ‘crt_copysigntf’
warning: compiler_builtins@0.1.131:    33 |       a = crt_copysigntf(crt_isinf(a) ? 1 : 0, a);
warning: compiler_builtins@0.1.131: cc1: some warnings being treated as errors

error: failed to run custom build command for `compiler_builtins v0.1.131`


Caused by:
  process didn't exit successfully: `/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-std/release/build/compiler_builtins-0f7d440f8ffa0feb/build-script-build` (exit status: 1)
  --- stdout
  cargo::rerun-if-changed=build.rs
  cargo::rerun-if-changed=configure.rs
  cargo::rustc-check-cfg=cfg(__ashldi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__ashlsi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__ashrdi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__ashrsi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__bswapsi2, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__bswapdi2, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__bswapti2, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__divdi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__divsi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__divmoddi4, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__divmodsi4, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__divmodsi4, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__divmodti4, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__lshrdi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__lshrsi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__moddi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__modsi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__muldi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__udivdi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__udivmoddi4, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__udivmodsi4, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__udivsi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__umoddi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__umodsi3, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas1_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas1_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas1_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas1_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas2_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas2_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas2_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas2_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas4_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas4_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas4_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas4_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas8_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas8_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas8_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas8_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas16_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas16_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas16_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_cas16_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd1_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd1_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd1_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd1_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd2_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd2_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd2_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd2_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd4_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd4_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd4_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd4_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd8_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd8_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd8_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldadd8_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr1_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr1_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr1_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr1_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr2_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr2_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr2_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr2_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr4_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr4_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr4_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr4_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr8_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr8_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr8_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldclr8_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor1_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor1_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor1_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor1_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor2_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor2_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor2_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor2_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor4_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor4_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor4_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor4_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor8_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor8_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor8_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldeor8_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset1_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset1_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset1_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset1_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset2_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset2_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset2_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset2_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset4_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset4_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset4_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset4_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset8_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset8_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset8_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_ldset8_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp1_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp1_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp1_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp1_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp2_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp2_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp2_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp2_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp4_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp4_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp4_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp4_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp8_relax, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp8_acq, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp8_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(__aarch64_swp8_acq_rel, values("optimized-c"))
  cargo::rustc-check-cfg=cfg(target_feature, values("vis3"))
  cargo::rustc-check-cfg=cfg(feature, values("checked"))
  cargo::rustc-check-cfg=cfg(assert_no_panic)
  cargo::rustc-check-cfg=cfg(f16_enabled)
  cargo::rustc-check-cfg=cfg(f128_enabled)
  cargo::rustc-cfg=f16_enabled
  cargo::rustc-cfg=f128_enabled
  cargo:compiler-rt=/cargo/registry/src/index.crates.io-6f17d22bba15001f/compiler_builtins-0.1.131/compiler-rt
  cargo::rustc-check-cfg=cfg(feature, values("unstable", "force-soft-floats"))
  cargo:rustc-cfg=feature="unstable"
  cargo:rustc-cfg=feature="force-soft-floats"
  cargo::rustc-check-cfg=cfg(feature, values("mem-unaligned"))
  cargo:rustc-cfg=feature="mem-unaligned"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/aarch64/lse.S
  cargo:rustc-cfg=__aarch64_cas1_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_cas1_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_cas1_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_cas1_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_cas2_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_cas2_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_cas2_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_cas2_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_cas4_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_cas4_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_cas4_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_cas4_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_cas8_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_cas8_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_cas8_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_cas8_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_cas16_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_cas16_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_cas16_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_cas16_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_swp1_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_swp1_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_swp1_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_swp1_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_swp2_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_swp2_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_swp2_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_swp2_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_swp4_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_swp4_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_swp4_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_swp4_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_swp8_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_swp8_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_swp8_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_swp8_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd1_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd1_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd1_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd1_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd2_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd2_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd2_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd2_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd4_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd4_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd4_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd4_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd8_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd8_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd8_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldadd8_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr1_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr1_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr1_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr1_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr2_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr2_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr2_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr2_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr4_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr4_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr4_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr4_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr8_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr8_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr8_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldclr8_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor1_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor1_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor1_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor1_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor2_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor2_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor2_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor2_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor4_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor4_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor4_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor4_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor8_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor8_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor8_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldeor8_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset1_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset1_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset1_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset1_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset2_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset2_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset2_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset2_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset4_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset4_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset4_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset4_acq_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset8_relax="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset8_acq="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset8_rel="optimized-c"
  cargo:rustc-cfg=__aarch64_ldset8_acq_rel="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/cpu_model/aarch64.c
  cargo:rustc-cfg=__aarch64_have_lse_atomics="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/absvdi2.c
  cargo:rustc-cfg=__absvdi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/absvsi2.c
  cargo:rustc-cfg=__absvsi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/absvti2.c
  cargo:rustc-cfg=__absvti2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/addvdi3.c
  cargo:rustc-cfg=__addvdi3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/addvsi3.c
  cargo:rustc-cfg=__addvsi3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/addvti3.c
  cargo:rustc-cfg=__addvti3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/cmpdi2.c
  cargo:rustc-cfg=__cmpdi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/cmpti2.c
  cargo:rustc-cfg=__cmpti2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/comparetf2.c
  cargo:rustc-cfg=__comparetf2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/divdc3.c
  cargo:rustc-cfg=__divdc3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/divsc3.c
  cargo:rustc-cfg=__divsc3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/fp_mode.c
  cargo:rustc-cfg=__fe_getround="optimized-c"
  cargo:rustc-cfg=__fe_raise_inexact="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/ffsti2.c
  cargo:rustc-cfg=__ffsti2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/floatditf.c
  cargo:rustc-cfg=__floatditf="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/floatsitf.c
  cargo:rustc-cfg=__floatsitf="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/floatunditf.c
  cargo:rustc-cfg=__floatunditf="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/floatunsitf.c
  cargo:rustc-cfg=__floatunsitf="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/int_util.c
  cargo:rustc-cfg=__int_util="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/muldc3.c
  cargo:rustc-cfg=__muldc3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/mulsc3.c
  cargo:rustc-cfg=__mulsc3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/multc3.c
  cargo:rustc-cfg=__multc3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/mulvdi3.c
  cargo:rustc-cfg=__mulvdi3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/mulvsi3.c
  cargo:rustc-cfg=__mulvsi3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/mulvti3.c
  cargo:rustc-cfg=__mulvti3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/negdf2.c
  cargo:rustc-cfg=__negdf2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/negdi2.c
  cargo:rustc-cfg=__negdi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/negsf2.c
  cargo:rustc-cfg=__negsf2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/negti2.c
  cargo:rustc-cfg=__negti2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/negvdi2.c
  cargo:rustc-cfg=__negvdi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/negvsi2.c
  cargo:rustc-cfg=__negvsi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/negvti2.c
  cargo:rustc-cfg=__negvti2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/paritydi2.c
  cargo:rustc-cfg=__paritydi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/paritysi2.c
  cargo:rustc-cfg=__paritysi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/parityti2.c
  cargo:rustc-cfg=__parityti2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/popcountdi2.c
  cargo:rustc-cfg=__popcountdi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/popcountsi2.c
  cargo:rustc-cfg=__popcountsi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/popcountti2.c
  cargo:rustc-cfg=__popcountti2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/subvdi3.c
  cargo:rustc-cfg=__subvdi3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/subvsi3.c
  cargo:rustc-cfg=__subvsi3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/subvti3.c
  cargo:rustc-cfg=__subvti3="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/ucmpdi2.c
  cargo:rustc-cfg=__ucmpdi2="optimized-c"
  cargo:rerun-if-changed=/checkout/src/llvm-project/compiler-rt/lib/builtins/ucmpti2.c
  cargo:rustc-cfg=__ucmpti2="optimized-c"
  OUT_DIR = Some(/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-std/aarch64-unknown-linux-musl/release/build/compiler_builtins-5ff9e91f8753f0ff/out)
  TARGET = Some(aarch64-unknown-linux-musl)
  HOST = Some(x86_64-unknown-linux-gnu)
  cargo:rerun-if-env-changed=CC_aarch64-unknown-linux-musl
  CC_aarch64-unknown-linux-musl = None
  cargo:rerun-if-env-changed=CC_aarch64_unknown_linux_musl
  cargo:rerun-if-env-changed=CC_aarch64_unknown_linux_musl
  CC_aarch64_unknown_linux_musl = Some(sccache aarch64-linux-musl-gcc)
  cargo:rerun-if-env-changed=CC_KNOWN_WRAPPER_CUSTOM
  CC_KNOWN_WRAPPER_CUSTOM = None
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  CRATE_CC_NO_DEFAULTS = None
  DEBUG = Some(true)
  CARGO_CFG_TARGET_FEATURE = Some(neon)
  cargo:rerun-if-env-changed=CFLAGS_aarch64-unknown-linux-musl
  cargo:rerun-if-env-changed=CFLAGS_aarch64-unknown-linux-musl
  CFLAGS_aarch64-unknown-linux-musl = None
  cargo:rerun-if-env-changed=CFLAGS_aarch64_unknown_linux_musl
  CFLAGS_aarch64_unknown_linux_musl = Some(-ffunction-sections -fdata-sections -fPIC)
  cargo:rerun-if-env-changed=CC_SHELL_ESCAPED_FLAGS
  CC_SHELL_ESCAPED_FLAGS = None
  OUT_DIR = Some(/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-std/aarch64-unknown-linux-musl/release/build/compiler_builtins-5ff9e91f8753f0ff/out)
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some(neon)
  cargo:rerun-if-env-changed=CFLAGS_aarch64-unknown-linux-musl
  CFLAGS_aarch64-unknown-linux-musl = None
  CFLAGS_aarch64-unknown-linux-musl = None
  cargo:rerun-if-env-changed=CFLAGS_aarch64_unknown_linux_musl
  CFLAGS_aarch64_unknown_linux_musl = Some(-ffunction-sections -fdata-sections -fPIC)
  cargo:rerun-if-env-changed=CC_SHELL_ESCAPED_FLAGS
  CC_SHELL_ESCAPED_FLAGS = None
  OUT_DIR = Some(/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-std/aarch64-unknown-linux-musl/release/build/compiler_builtins-5ff9e91f8753f0ff/out)
  cargo:rerun-if-env-changed=CC_ENABLE_DEBUG_OUTPUT
  CRATE_CC_NO_DEFAULTS = None
  CARGO_CFG_TARGET_FEATURE = Some(neon)
  cargo:rerun-if-env-changed=CFLAGS_aarch64-unknown-linux-musl
  CFLAGS_aarch64-unknown-linux-musl = None
  CFLAGS_aarch64-unknown-linux-musl = None
  cargo:rerun-if-env-changed=CFLAGS_aarch64_unknown_linux_musl
  CFLAGS_aarch64_unknown_linux_musl = Some(-ffunction-sections -fdata-sections -fPIC)
  cargo:rerun-if-env-changed=CC_SHELL_ESCAPED_FLAGS
  CC_SHELL_ESCAPED_FLAGS = None
  cargo:warning=In file included from /checkout/src/llvm-project/compiler-rt/lib/builtins/fp_lib.h:24,
  cargo:warning=                 from /checkout/src/llvm-project/compiler-rt/lib/builtins/multc3.c:14:
  cargo:warning=/checkout/src/llvm-project/compiler-rt/lib/builtins/multc3.c: In function ‘__multc3’:
  cargo:warning=/checkout/src/llvm-project/compiler-rt/lib/builtins/int_math.h:71:32: error: implicit declaration of function ‘__builtin_copysignq’; did you mean ‘__builtin_copysign’? [-Werror=implicit-function-declaration]
  cargo:warning=   71 | #define crt_copysignf128(x, y) __builtin_copysignq((x), (y))
  cargo:warning=      |                                ^~~~~~~~~~~~~~~~~~~
  cargo:warning=/checkout/src/llvm-project/compiler-rt/lib/builtins/fp_lib.h:394:24: note: in expansion of macro ‘crt_copysignf128’
  cargo:warning=  394 | #define crt_copysigntf crt_copysignf128
  cargo:warning=      |                        ^~~~~~~~~~~~~~~~
  cargo:warning=/checkout/src/llvm-project/compiler-rt/lib/builtins/multc3.c:33:11: note: in expansion of macro ‘crt_copysigntf’
  cargo:warning=   33 |       a = crt_copysigntf(crt_isinf(a) ? 1 : 0, a);
  cargo:warning=      |           ^~~~~~~~~~~~~~
  cargo:warning=cc1: some warnings being treated as errors
  --- stderr



  error occurred: Command "sccache" "aarch64-linux-musl-gcc" "-O3" "-ffunction-sections" "-fdata-sections" "-fPIC" "-gdwarf-4" "-fno-omit-frame-pointer" "-I" "/checkout/src/llvm-project/compiler-rt/lib/builtins" "-ffunction-sections" "-fdata-sections" "-fPIC" "-fno-builtin" "-fvisibility=hidden" "-ffreestanding" "-Werror=implicit-function-declaration" "-fomit-frame-pointer" "-ffile-prefix-map=/checkout/src/llvm-project/compiler-rt=." "-DCOMPILER_RT_HAS_FLOAT16" "-DVISIBILITY_HIDDEN" "-o" "/checkout/obj/build/x86_64-unknown-linux-gnu/stage1-std/aarch64-unknown-linux-musl/release/build/compiler_builtins-5ff9e91f8753f0ff/out/45c91108d938afe8-multc3.o" "-c" "/checkout/src/llvm-project/compiler-rt/lib/builtins/multc3.c" with args aarch64-linux-musl-gcc did not execute successfully (status code exit status: 1).

warning: build failed, waiting for other jobs to finish...
[RUSTC-TIMING] core test:false 17.717
Build completed unsuccessfully in 0:32:38
```

</details>

--- a/library/alloc/Cargo.toml
+++ b/library/alloc/Cargo.toml
@@ -10,7 +10,7 @@ edition = "2021"
 
 [dependencies]
 core = { path = "../core" }
-compiler_builtins = { version = "0.1.130", features = ['rustc-dep-of-std'] }
+compiler_builtins = { version = "0.1.132", features = ['rustc-dep-of-std'] }
 
 [dev-dependencies]
 rand = { version = "0.8.5", default-features = false, features = ["alloc"] }

--- a/library/std/Cargo.toml
+++ b/library/std/Cargo.toml
@@ -17,7 +17,7 @@ cfg-if = { version = "1.0", features = ['rustc-dep-of-std'] }
 panic_unwind = { path = "../panic_unwind", optional = true }
 panic_abort = { path = "../panic_abort" }
 core = { path = "../core", public = true }
-compiler_builtins = { version = "0.1.130" }
+compiler_builtins = { version = "0.1.132" }
 profiler_builtins = { path = "../profiler_builtins", optional = true }
 unwind = { path = "../unwind" }
 hashbrown = { version = "0.15", default-features = false, features = [

